### ISSUE TITLE
Receiver Facing: Error says CDC Prime - Now what?

### ISSUE THREAD

**Issue by victorialb-usds** [CDCgov/prime-reportstream#4581]
As a user that receives COVID testing data from RS via the CSV upload pilot feature, 
When my rhapsody system kicks out an error, and it says CDC Prime, I do not know the facility sending the information
Who and how will the submitter be contacted to fix the data. 

Problem: 
- it is not clear who is responsible for contacting the submitter 

Background: 
This came up in interim check in call with Colorado PHE
https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/_layouts/15/Doc.aspx?sourcedoc=%7B9571319D-5BD9-4B42-AECE-B1E39443B8DB%7D&file=COPHE_Interim_CSV%20Pilot%20Upload%20Program_2.23.22.docx&action=default&mobileredirect=true

[Slack thread](https://slack.com/archives/C025F5W5Q7L/p1646669126890549)

TO DO: 
- f/u with COPHE to let them know we are working on this - we said we would look into it on 2/23
- get more info from OO team on current state

**Comment by rachelhanster** [comment]
@victorialb-usds @rhood23699 Who from Colorado asked the initial question? Would the following solve Colorado's question?
Look at Reporting Facility name in MSH.4. In cases where a centralized name and CLIA are used in the MSH.4 (ie, CA-DPH), the PHD can look at OBX.23 Performing/Testing Facility.

**Comment by victorialb-usds** [comment]
It was first asked by Alyson via email and then it was asked again by Nancy.

________________________________

**Comment by rachelhanster** [comment]
@BerniXiongA6 to reach out to @Adrian-Brewster so someone from Ops team can respond to Colorado's question.
Please also provide a recommendation for how customer support questions are to be captured internally if/when they come up during a research session.
cc @victorialb-usds

**Comment by kjt9cc** [comment]
can we CC or soft hand this off to the reporstream@cdc.gov inbox when questions like this come in?

**Comment by Adrian-Brewster** [comment]
I emailed COPHE (CC'd Victoria) with an explanation of how to identify the sending facility for a message that has errored out.  

When a message fails or NACKS at the state surveillance system, they should contact Reportstream@cdc.gov with the affected files message ID or accession number and the O&O team will investigate the cause of failure and work with the sender to correct and resubmit.  This takes the burden off of the state for chasing down issues with the sender.  This the data is so heavily validated before moving to production, this has not been an issue thus far (knocking on wood).  I explained this to Nancy in my email as well.  

thank you for capturing the new workflow @BerniXiongA6 . I think we can go ahead and close this out now.
